sparkle/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dust-tt/sparkle",
-  "version": "0.2.151",
+  "version": "0.2.159",
```

Comment:
Why such a bump?

Reply:
I probably got carried away before understanding how suffix works on the "public alpha" action. Fixed to the next incremental version (0.2.150) 🙏